spec/src/main/asciidoc/histogram-timer-config.adoc

```diff
@@ -0,0 +1,231 @@
+
+[[histogram-bucket-sample]]
+.MicroProfile Config `mp.metrics.distribution.histogram.buckets` property examples
+----
+//alpha.histogram will publish histogram-buckets with maximum values of 10.0, 50.0 and 100.0 while the beta.histogram will publish histogram buckets with maximum values of 30.0, 50 and 123.
+mp.metrics.distribution.histogram.buckets=alpha.histogram=10.0,50.0,100.0;beta.histogram=30.0,50.0,123
+
+//any histogram that matches with `beta.*` will publish histograms buckets with maximum values of 50.0 and 100.0 except the beta.test.histogram which will publish a bucket with a maximum value of 100.0 due to precedence.
+mp.metrics.distribution.histogram.buckets=beta.*=50.0,100.0;beta.test.histogram=100.0
+
+----
+
+NOTE: Histograms that do not match the metric name definitions of the `mp.metrics.distribution.histogram.buckets` property will output no buckets.
+
+The following is a sample output of the `alpha.histogram` defined in the first `mp.metrics.distribution.histogram.buckets` definition above.
+
+[[histogram-bucket-output]]
+.Sample alpha.histogram Prometheus output
+----
+# HELP alpha_histogram_cookies  
+# TYPE alpha_histogram_cookies histogram <1>
+alpha_histogram_cookies{mp_scope="application",quantile="0.5",} 9.25 <2>
+alpha_histogram_cookies{mp_scope="application",quantile="0.75",} 51.75 <2>
+alpha_histogram_cookies{mp_scope="application",quantile="0.95",} 51.75 <2>
+alpha_histogram_cookies{mp_scope="application",quantile="0.98",} 51.75 <2>
+alpha_histogram_cookies{mp_scope="application",quantile="0.99",} 51.75 <2>
+alpha_histogram_cookies{mp_scope="application",quantile="0.999",} 51.75 <2>
+alpha_histogram_cookies_bucket{mp_scope="application",le="10.0",} 2.0 <3>
+alpha_histogram_cookies_bucket{mp_scope="application",le="50.0",} 3.0 <3>
+alpha_histogram_cookies_bucket{mp_scope="application",le="100.0",} 3.0 <3>
+alpha_histogram_cookies_bucket{mp_scope="application",le="+Inf",} 3.0 <3><4>
+alpha_histogram_cookies_count{mp_scope="application",} 3.0
+alpha_histogram_cookies_sum{mp_scope="application",} 64.0
+# HELP alpha_histogram_cookies_max  
+# TYPE alpha_histogram_cookies_max gauge
+alpha_histogram_cookies_max{mp_scope="application",} 50.0
+----
+<1> The Prometheus metric type is `histogram`
+
+<2> percentiles are part of  the `histogram` type instead of `summary`
+
+<3> histogram buckets
+
+<4> The `+Inf` bucket is included in the output
+
+
+
+[[timer-buckets-sample]]
+.MicroProfile Config `mp.metrics.distribution.timer.buckets` property examples
+----
+//alpha.timer will publish histogram-buckets with maximum values of 500 milliseconds, 2 seconds, and 3 minutes while the beta.timer will publish histogram buckets with maximum values of 10 seconds, 2 minutes and 5 hours. Timer metrics that do not match will not have any histogram buckets.
+mp.metrics.distribution.timer.buckets=alpha.timer=10,500ms,2s,3m;beta.timer=10s,2m,5h
```

Comment:
The setting for `alpha.timer` specifies `10` first, but the command describing the setting does not mention it.

Reply:
oh, fixed!